handler/src/main/java/io/netty5/handler/traffic/AbstractTrafficShapingHandler.java

```diff
@@ -588,11 +588,14 @@ public void channelRegistered(ChannelHandlerContext ctx) throws Exception {
         ctx.fireChannelRegistered();
     }
 
+    // TODO: Fix me later!
```

Comment:
@chrisvest same comment... I think we need to think about a better way how to support something like this. Maybe sending an event to let the transport know about this.

Reply:
It seems niche enough that an event sounds like the right approach.